modules/publicize/publicize-jetpack.php

```diff
@@ -447,7 +447,11 @@ function test_connection( $service_name, $connection ) {
 	 * Save a flag locally to indicate that this post has already been Publicized via the selected
 	 * connections.
 	 */
-	function save_publicized( $post_ID, $post, $update ) {
+	function save_publicized( $post_ID = null, $post = null, $update = null ) {
```

Comment:
The original ticket only has warnings for arguments 2 and 3, so I suspect those are the only ones that need defaults and checks. As per Eric's message below, just having a default value for `$update` suppresses the warning properly, so there's no need to check its actual value.

This might be more concise: 

```php
function save_publicized( $post_ID, $post = null, $update = null ) {
  if ( is_null( $post ) ) {
    return;
  }
}
```